FreeIT/src/lesson_10/Application.java

```diff
@@ -0,0 +1,237 @@
+package lesson_10;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Scanner;
+
+public class Application {
+
+	private Library libr;
+
+	public Application() {
+		super();
+		ArrayList<Book> book = new ArrayList<>();
+		Book book1 = new Book(123, "Atlant", "Roman");
+		Book book2 = new Book(345, "Puaro", "Detective");
+		Book book3 = new Book(567, "NLO", "Fantasy");
+		book.add(book1);
+		book.add(book2);
+		book.add(book3);
+		Library lib = new Library(book);
+		libr = lib;
+	}
+
+	public void start() {
```

Comment:
why do you need method that just calls another method?

Reply:
To do the recurtion in just another method)

Reply:
It's very very bad solution